codes/kotlin/chapter_dynamic_programming/climbing_stairs_dfs_mem.kt

```diff
@@ -0,0 +1,37 @@
+/**
+ * File: climbing_stairs_dfs_mem.kt
+ * Created Time: 2024-01-25
+ * Author: curtishd (1023632660@qq.com)
+ */
+
+package chapter_dynamic_programming
```

Comment:
You should remove the original file if you change the file name directly.

If you would like to change the file name with git, you can use `git mv` command.